app.py

```diff
@@ -71,6 +82,18 @@ class status(Enum):
 )
 
 
+def _calculate_chunk_size(num_roles: int) -> int:
+    chunk_size_cfg = repository._settings.get_fresh(
+        "BUMP_ONLINE_ROLES_CHUNK_SIZE", 500
+    )
+    if num_roles <= chunk_size_cfg:
+        chunk_size = int(num_roles / 2)
```

Comment:
What if chunk_size_cfg is triple larger than num_roles?
For example: chunk_size = 50000, num_roles = 2000.
Should we have a more precise calculation here to make sure chunk_size will be indeed lower than num_roles?
Not a blocker, just a comment.

Reply:
Ok, I will improve it.

Reply:
Hi @MVrachev 
Yesterday evening I refactored it and added full UT for the tasks (forgot to push)
https://github.com/repository-service-tuf/repository-service-tuf-worker/pull/689/commits/204158745e0058bc3dc37d06f5e42509bbcc9086